sumologic/resource_sumologic_dashboard_test.go

```diff
@@ -40,7 +40,7 @@ func TestAccSumologicDashboard_create(t *testing.T) {
 	title := "terraform_test_dashboard_" + testNameSuffix
 	description := "Test dashboard description"
 	theme := "Dark"
-	refreshInterval := 30
```

Comment:
Why is 30 an invalid value for `refreshInterval`?

Reply:
Not sure, that's just what the error message says.

Reply:
`Please use from the following list: 120,300,900,1800,3600,7200,86400.`
Is this a breaking change?

Reply:
Yeah, if we only allow certain values, this is a breaking API change.